examples/lm1b_nnx/main.py

```diff
@@ -34,7 +34,7 @@
   'File path to the training hyperparameter configuration.',
   lock_config=True,
 )
-flags.mark_flags_as_required(['config', 'workdir'])
+flags.mark_flags_as_required(['workdir'])
```

Comment:
This fixes a warning saying that config flag has a default value so it can't be strictly required.